### PR TITLE
Added missing note about Gradle wrapper required

### PR DESCRIPTION
**Pull request by wild-lynx**
...for the CocoaPods plugin.

### CODE REVIEW

File: COCOAPODS.md [hunk 1]
```diff
@@ -94,6 +94,11 @@ file for the project.
 The `Podspec` file includes a path to an output framework and script phases that automate building this framework during 
 the build process of an Xcode project.
 
+6. Generate Gradle wrapper. In order to avoid compatibility issues during an Xcode build, the plugin requires using
```
Comment (katia-energizer) [inline COCOAPODS.md:97]: I would rephrase like this:

> 6. Generate the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html), which helps avoid compatibility issues during an Xcode build.

**Comment by koshachy** [comment]
Fixed in #4507
